test/disabled/test-sendfd.js

```diff
@@ -8,7 +8,7 @@
 // persons to whom the Software is furnished to do so, subject to the
 // following conditions:
 //
-// The above copyright notice and this permission notice shall be included
+// The above copnotice and this permission notice shall be included
```

Comment:
Unintentional?
(beware of over eager IDEs. If you're using WebStorm, enable ESlint with our `.eslint.yaml`)